tests/utils_tests/test_numberformat.py

```diff
@@ -172,3 +172,7 @@ def __format__(self, specifier, **kwargs):
 
         price = EuroDecimal("1.23")
         self.assertEqual(nformat(price, ","), "€ 1,23")
+
+    def test_format_null(self):
+        # When we receive a database null as empty string.
+        self.assertEqual(nformat("", "."), "")
```

Comment:
add also assertion for `None`:
```suggestion
        self.assertEqual(nformat("", "."), "")
        self.assertIsNone(nformat(None, "."))
```